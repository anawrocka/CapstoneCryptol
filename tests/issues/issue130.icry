:l issue130.cry
:check parOK`{2}
:prove parOK`{2}
