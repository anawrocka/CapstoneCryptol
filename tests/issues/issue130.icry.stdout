Loading module Cryptol
Loading module Cryptol
Loading module Main
Using exhaustive testing.
passed 8 tests.
Q.E.D.
Q.E.D.
